src/port1.0/portutil.tcl

```diff
@@ -508,6 +508,16 @@ proc default_check {optionName index op} {
     }
 }
 
+# env_default 
+# Unless port is Xcode dependent, returns env which sets developer_dir to Command Line Tools
+proc env_default {} {
```

Comment:
@mojca I'll have to investigate that, but we can put some conditionals here if the path's different across versions.